app/controllers/application_controller/ci_processing.rb

```diff
@@ -2508,6 +2508,18 @@ def providehosts
     host_button_operation('provide', _('Provide'))
   end
 
+  def refreshphysicalservers
+    # TODO: refresh physical servers
+  end
+
+  def deletephysicalservers
+    # TODO: delete physical servers
+  end
+
+  def editphysicalservers
+    # TODO: edit physical servers
+  end
+
```

Comment:
Please move these methods in `physical_server_controller`.
Also rename the methods to use snake_case (i.e. `refresh_physical_server` as opposed to `refreshphysicalserver`)